megamek/src/megamek/client/ui/swing/unitSelector/TWAdvancedSearchPanel.java

```diff
@@ -445,6 +451,8 @@ private JPanel createBasePanel() {
         loadTriStateItem(Mech.COCKPIT_STRING, listCockpitType, 7);
         loadTriStateItem(EquipmentType.structureNames, listInternalsType, 7);
         loadTriStateItem(Engine.getEngineTypes(), listEngineType, 5);
+        loadTriStateItem(SimpleTechLevel.getDescriptions(), listTechLevel, 5);
```

Comment:
I have noticed in a previous PR but didn't comment on it. Would you mind adding a version of this that ccepts a List<String>. Lists are generally preferable to arrays for safety and general ease of handling and well, modernity. Generating that list im SimpleTechLevel is at least one method call shorter then. The getEngineTypes method would also be simpler.

Reply:
a `List<String>`
the formatting swallowed that :)